### PR TITLE
Feature/testnet script

### PR DESCRIPTION
**Pull request by nbailo**

### CODE REVIEW

**Review by Aksonov-Artem — APPROVED** [review]

File: contracts/test/THybridToken.sol [hunk 1]
```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: Apache-2.0
+
+pragma solidity >=0.6.6;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+
+contract THybridToken is ERC20 {
```
Comment (a-melnichuk) [inline contracts/test/THybridToken.sol]: THybridToken -> TestHybridToken

File: contracts/test/TIndexHybridToken.sol [hunk 2]
```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: Apache-2.0
+
+pragma solidity >=0.6.6;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "@openzeppelin/contracts/utils/ReentrancyGuard.sol";
+import "../IndexHybridToken.sol";
+
+
+contract TIndexHybridToken is IndexHybridToken {
```
Comment (a-melnichuk) [inline contracts/test/TIndexHybridToken.sol]: TIndexHybridToken -> TestIndexHybridToken

File: contracts/test/TSaleHybridToken.sol [hunk 3]
```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: Apache-2.0
+
+pragma solidity >=0.6.6;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "@openzeppelin/contracts/utils/ReentrancyGuard.sol";
+import "../SaleHybridToken.sol";
+
+
+contract TSaleHybridToken is SaleHybridToken {
```
Comment (a-melnichuk) [inline contracts/test/TSaleHybridToken.sol]: TSaleHybridToken -> TestSaleHybridToken

File: contracts/test/TVestingSwap.sol [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/utils/ReentrancyGuard.sol";
+import "../../interfaces/ISaleHybridToken.sol";
+import "../../interfaces/IPresale.sol";
+import "../../libraries/SafeTransfer.sol";
+
+
+
+contract TVestingSwap is Ownable, ReentrancyGuard {
```
Comment (a-melnichuk) [inline contracts/test/TVestingSwap.sol]: TVestingSwap -> TestVestingSwap

**Review by a-melnichuk — CHANGES_REQUESTED** [review]

**Review by a-melnichuk — APPROVED** [review]

**Pull request merged**